TechDraw/TechDrawTools.FCMacro

```diff
@@ -70,6 +79,147 @@ class LineAttributes:
     def getWidth(self):
         return self.LineWidth
 
+class translate:
+    '''do all translations'''
+    def __init__(self):
+        self.langKey = self.getLanguage()
+
+    def getLanguage(self):
+        '''get local language key'''
+        langDict = {}
+        langDict["English"] = 0
+        langDict["German"] = 1
+        langDict["Italian"] = 2
+        langDict["French"] = 3
+        langDict["Spanish"] = 4
+        self.langKey = 0
+        if App.Gui.getLocale() in langDict.keys():
+            self.langKey = langDict[App.Gui.getLocale()]
```

Comment:
I don't see the interest of using integers in `langDict`. Why don't you use directly `self.langKey = 'English'` or `self.langDict = 'English'`?